api/snapshot.go

```diff
@@ -182,6 +182,8 @@ func (s *Server) SnapshotBackup(w http.ResponseWriter, req *http.Request) (err e
 		labels[types.KubernetesStatusLabel] = string(kubeStatus)
 	}
 
+	labels[types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode)] = string(vol.Spec.AccessMode)
+
```

Comment:
Suggest moving to the mutating webhook.

Reply:
Thanks, I will follow the suggestion to update access mode value into backup label through mutating webhook `webhook/resources/volume/mutator.go`.